skimage/_shared/utils.py

```diff
@@ -19,6 +20,54 @@ class skimage_deprecation(Warning):
     pass
 
 
+class change_default_value:
+    """Decorator for changing the default value of an argument.
+
+    Parameters
+    ----------
+    arg_name: str
+        The name of the argument to be updated.
+    old_value: any
+        The argument old value.
+    warning_msg: str
```

Comment:
I feel like this should come after `changed_version`, which is not optional (or should not be).

Reply:
Indeed, I just saw that in the actual argument list, it comes later, as expected.

Reply:
... and old_value is not a parameter!

Reply:
`old_value` comes from previous commits :smile: I will fix it, thank you